cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/nosql/NoSqlTransactionRunner.java

```diff
@@ -50,13 +58,17 @@ public NoSqlTransactionRunner(NoSqlStructuredTableAdmin tableAdmin, TransactionS
         }
       }),
       RetryStrategies.retryOnConflict(20, 100));
```

Comment:
Transaction conflict metrics should be added here

Reply:
I think the metrics is emitted in Tephra for nosql case, right?

Reply:
File a JIRA to verify later then

Reply:
Opened https://issues.cask.co/browse/CDAP-15055